.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -39,6 +39,23 @@ body:
         - [Minimal Complete Verifiable Examples](https://stackoverflow.com/help/mcve)
         - [Craft Minimal Bug Reports](https://matthewrocklin.com/minimal-bug-reports)
 
+        Please list dependencies in [inline script metadata](https://packaging.python.org/en/latest/specifications/inline-script-metadata/#example).
+        When put in a file named `issue.py` calling `uv run issue.py` should show the issue.
```

Comment:
I would make this a little weaker, maybe "Consider listing" instead of "Please list". Not everyone has `uv` installed, e.g., if they are running Xarray from a managed Python environment like Google Colab.